### PR TITLE
front end page designs

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

**Comment by KaitCrawford** [comment]
@miltontony ready for review. just waiting on the twitter url

File: unicorecmsmnmcameroon/templates/base.pt [hunk 1]
```diff
@@ -18,27 +17,19 @@
...
-    </div>
 
     <div id="content">
         <div metal:define-slot="content"></div>
     </div>
-    <div id="base"><img alt='' src="/static/img/ConnectSmart_UNICEF-logo-footer.png" /></div><!--/BASE-->
+    <div id="base">
+            Copyright &copy;2014<br/>
+            Malaria No More<br/>
+            <a href="/contact/" i18n:translate="Contact">Contact Us</a>
```
Comment (miltontony) [inline unicorecmsmnmcameroon/templates/base.pt]: The translate string should be the same as the string i.e `i18n:translate="Contact Us"`

File: unicorecmsmnmcameroon/templates/home.pt [hunk 2]
```diff
@@ -22,22 +22,38 @@
...
                 </condition>
-                <div class="h2"><a href="/content/detail/${page.uuid}/">${page.title}</a></div>
                 <p>${page.description}</p>
                 <div class="clr"></div>
             </div>
 
             <div class="clr"></div>
             <div class="pagination more category-${cat.slug}" tal:condition="len(view.get_featured_category_pages(cat.uuid)) > 1">
-                <a href="/content/list/${cat.uuid}/"><span i18n:translate="More">More</span> &gt;</a>
+                <a href="/content/list/${cat.uuid}/"><span i18n:translate="More">Read more posts</span></a>
```
Comment (miltontony) [inline unicorecmsmnmcameroon/templates/home.pt]: Lets just keep this as "read more" and we'll inform them about it

File: unicorecmsmnmcameroon/templates/home.pt [hunk 3]
```diff
@@ -22,22 +22,38 @@
...
             </div>
         </div>
 
-        <div class="declaration">
-            <div class="pagination"><a href="/contact/" class="contact" i18n:translate="Contact">Contact</a> | <a href="/credits/" class="terms" i18n:translate="Credits">Credits</a> | <a href="/about/" class="terms" i18n:translate="About">About</a></div>
-        </div><!--/declaration-->
+        <div class="articles">
+            <div class="h1">Connect With Us</div>
```
Comment (miltontony) [inline unicorecmsmnmcameroon/templates/home.pt]: i18n missing

File: unicorecmsmnmcameroon/templates/home.pt [hunk 4]
```diff
@@ -22,22 +22,38 @@
...
+            <div class="links">
+                <a href="http://mnm-cm-185.apollo.unicore.io"><img src="/static/img/facebook-logo.png">Facebook</a>
```
Comment (miltontony) [inline unicorecmsmnmcameroon/templates/home.pt]: This URL looks suspiciously wrong

File: unicorecmsmnmcameroon/templates/home.pt [hunk 5]
```diff
@@ -22,22 +22,38 @@
...
+
+                <a href="https://www.malarianomore.org/country/cameroon"><img src="/static/img/nmcp-logo.png">NMCP Website</a>
+            </div>
+        </div>
+        
+        <div class="articles">
+            <div class="h1">Our Supporters</div>
+            <div class="listing">
+                <div id="thanks">
+                    <p>We'd like to thank ExxonMobil, the MTN Foundation, the Clinton Foundation and Peace Corps</p>
```
Comment (miltontony) [inline unicorecmsmnmcameroon/templates/home.pt]: You'll probably need to i18n this too

File: unicorecmsmnmcameroon/templates/home.pt [hunk 6]
```diff
@@ -22,22 +22,38 @@
...
+                <a href="http://mnm-cm-185.apollo.unicore.io"><img src="/static/img/facebook-logo.png">Facebook</a>
+
+                <a href="#"><img src="/static/img/twitter-logo.png">Twitter</a>
+
+                <a href="https://www.malarianomore.org/country/cameroon"><img src="/static/img/nmcp-logo.png">NMCP Website</a>
+            </div>
+        </div>
+        
+        <div class="articles">
+            <div class="h1">Our Supporters</div>
```
Comment (miltontony) [inline unicorecmsmnmcameroon/templates/home.pt]: i18n missing

File: unicorecmsmnmcameroon/templates/home.pt (hunk 4 above)
Comment (KaitCrawford) [inline unicorecmsmnmcameroon/templates/home.pt]: How so? This is the one Wouda gave me and it seems to work

Comment (miltontony) [inline unicorecmsmnmcameroon/templates/home.pt]: Isn't it supposed to be a facebook link? It currently links to an existing Malaria No More instance

File: unicorecmsmnmcameroon/templates/home.pt (hunk 5 above)
Comment (KaitCrawford) [inline unicorecmsmnmcameroon/templates/home.pt]: Does the translation string have to match the string here as well?

File: unicorecmsmnmcameroon/templates/home.pt (hunk 4 above)
Comment (KaitCrawford) [inline unicorecmsmnmcameroon/templates/home.pt]: I did check this with her, she requested that the facebook links point to the i.org sites

File: unicorecmsmnmcameroon/templates/home.pt (hunk 5 above)
Comment (miltontony) [inline unicorecmsmnmcameroon/templates/home.pt]: Yes, it does.

**Comment by KaitCrawford** [comment]
@miltontony ready for final review

**Comment by miltontony** [comment]
:+1: looks good

**Pull request merged**